kong/runloop/handler.lua

```diff
@@ -1478,7 +1488,17 @@ return {
         end
 
         if enabled_headers[headers.VIA] then
```

Comment:
```suggestion
        if enabled_headers[headers.VIA] then
```

This is for downstream headers, and now we are using it for upstream headers too, so should we somehow adjust the documentation:

https://github.com/Kong/kong/blob/master/kong.conf.default#L897-L898

Or should we apply upstream header regardless of what in configured in `headers`?

Reply:
@bungle this part is only used for downstream Via header. According RFC, it is optionally.

The upstream Via header is enforced now, according to RFC.